Header header 
float64 exp_time    # ms
